requirements_all.txt

```diff
@@ -85,6 +85,9 @@ paho-mqtt==1.1
 # homeassistant.components.mqtt
 jsonpath-rw==1.4.0
 
+# homeassistant.components.mysensors
+https://github.com/theolind/pymysensors/archive/2aa8f32908e8c5bb3e5c77c5851db778f8635792.zip#pymysensors==0.3
```

Comment:
You'll need to version bump pymysensors because otherwise HA is unable to distinguish this and the previous version 0.3 and thus not install the new version.